eng/generators.targets

```diff
@@ -56,15 +56,18 @@
     <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.InteropServices\gen\Microsoft.Interop.SourceGeneration\Microsoft.Interop.SourceGeneration.csproj"
                       ReferenceOutputAssembly="false"
                       OutputItemType="Analyzer"
+                      SetPlatform="Platform=AnyCPU"
```

Comment:
Should we also set it for https://github.com/dotnet/runtime/blob/18bc1158575cc4c653a660d4a52c0217ea20c647/src/libraries/System.Private.CoreLib/gen/System.Private.CoreLib.Generators.csproj#L5?